camerata.gemspec

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require_relative 'lib/camerata/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "camerata"
+  spec.version       = Camerata::VERSION
+  spec.authors       = ["Rob Kaufman"]
+  spec.email         = ["rob@notch8.com"]
+
+  spec.summary       = 'Coordinate services for YUL-DC project'
+  spec.description   = 'Command line tools and other combined services for the YUL-DC project. Shared Docker and deployment configuration, CI tools, etc live here.'
+  spec.homepage      = "https://github.com/yalelibrary/yul-dc-camerata"
+  spec.license       = "MIT"
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+
+  spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
```

Comment:
Do you want to update this now, or later?

Reply:
right now it s"no where" so this is an ok setting